Example/StandardIntegrationUITests/StandardIntegrationUITests.swift

```diff
@@ -17,7 +17,7 @@ class StandardIntegrationUITests: XCTestCase {
         // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
         let app = XCUIApplication()
         let stripePublishableKey = "pk_test_6Q7qTzl8OkUj5K5ArgayVsFD00Sa5AHMj3"
-        let backendBaseURL = "https://stripe-mobile-test-backend.herokuapp.com/"
+        let backendBaseURL = "https://stripe-mobile-test-backend-17.herokuapp.com/"
```

Comment:
Hah, forgot that versioning would be an issues on these tests. This seems to be an okay workaround, but if it gets too toilsome we should figure out something else.